src/modules/fancyzones/lib/Zone.cpp

```diff
@@ -37,6 +38,20 @@ struct Zone : winrt::implements<Zone, IZone>
     std::map<HWND, RECT> m_windows{};
 };
 
+IFACEMETHODIMP_(bool) Zone::Focus() noexcept
+{
+    if (IsEmpty())
+    {
+        return false;
+    }
+    auto win = m_windows.begin()->first;
+    if (SetForegroundWindow(win))
+    {
```

Comment:
Not sure if it's the correct API to call.

Reply:
It won't work if the window is minimized, you first have to restore it.